src/main/pages/che-7/administration-guide/proc_collecting-che-metrics-with-prometheus.adoc

```diff
@@ -31,18 +33,10 @@ histogram:: a more complex metric that is suited for performing observations. Me
       scrape_configs:
         - job_name: 'che'
           static_configs:
-            - targets: ['che-host:8087']
+            - targets: ['{prod-host}:8087']
   kind: ConfigMap
   metadata:
     name: prometheus-config
 ----
 <1> rate, at which a target is scraped
 <2> rate, at which recording and alerting rules are re-checked (not used in our system at the moment)
-
-
-// [discrete]
-// == Additional resources
-// 
-// * A bulleted list of links to other material closely related to the contents of the procedure module.
-// * For more details on writing procedure modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
-// * Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
```

Comment:
//// 
.Additional resources
 
 * A bulleted list of links to other material closely related to the contents of the procedure module.
 * For more details on writing procedure modules, see the link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
 * Use a consistent system for file names, IDs, and titles. For tips, see _Anchor Names and File Names_ in link:https://github.com/redhat-documentation/modular-docs#modular-documentation-reference-guide[Modular Documentation Reference Guide].
////